### PR TITLE
 DRFT-710: Create admin endpoint on drift-backend

### PR DESCRIPTION
**Pull request by fellipeh**
This PR adds admin endpoint for stage test purpose

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: drift/views/admin.py
Comment (sthirugn) [inline drift/views/admin.py:13]: can you indicate somewhere that this is test data?

File: drift/app.py
Comment (CodeHeeler) [inline drift/app.py:27]: In this file, you will also need to import similar to here in baselines:
https://github.com/RedHatInsights/system-baseline-backend/blob/master/system_baseline/app.py#L15

And you will need to register the blueprint as well I think:
https://github.com/RedHatInsights/system-baseline-backend/blob/master/system_baseline/app.py#L56

See this PR for further comparisons of how this was done for the internal endpoint:
https://github.com/RedHatInsights/system-baseline-backend/pull/179/files

File: drift/openapi/admin_api.spec.yaml
Comment (CodeHeeler) [inline drift/openapi/admin_api.spec.yaml:15]: I feel like I recall the version being required, I don't know if there would be an exception for admin, but here is an example:
https://github.com/RedHatInsights/system-baseline-backend/blob/master/system_baseline/openapi/internal_api.spec.yaml#L14-L26

File: drift/openapi/admin_api.spec.yaml
Comment (CodeHeeler) [inline drift/openapi/admin_api.spec.yaml:19]: if so on my above comment about version, don't forget to add the version in here as well as `drift.admin_views.v1.status`

And I do think you need your own separate views like internal_views and mgmt_views, you would now have admin_views folder and v1.py within that.

File: drift/views/admin.py
Comment (CodeHeeler) [inline drift/views/admin.py:1]: Again, unless I am misunderstanding the new admin work, I think this file needs to be drift/admin_views/v1.py instead.

**Review by CodeHeeler — CHANGES_REQUESTED** [review]
I've left a number of comments on things that need changing, but everything else looks great!  Thank you so much for working on this.

File: drift/views/admin.py
Comment (CodeHeeler) [reply drift/views/admin.py:13]: yeah, adding a comment to that effect is a good idea

File: drift/openapi/admin_api.spec.yaml
Comment (fellipeh) [reply drift/openapi/admin_api.spec.yaml:15]: During the design process we decide to move the admin without any version, because admin will be used only by managers etc.. and will not have several changes

Comment (CodeHeeler) [reply drift/openapi/admin_api.spec.yaml:15]: Ok, in that case, if it works locally and passes automation and schema checks, then that's fine.

File: drift/views/admin.py
Comment (CodeHeeler) [reply drift/views/admin.py:1]: given your decision to not have version, then this is probably drift/admin_views/admin.py for the naming convention then?

File: drift/app.py
Comment (fellipeh) [reply drift/app.py:27]: fixed.

**Review by sthirugn — APPROVED** [review]
please wait for @CodeHeeler final review too

**Comment by sthirugn** [comment]
nitpick: it will be great to add jira id in commit message

**Review by CodeHeeler — APPROVED** [review]
LGTM, thanks for considering my suggestions.

**Review by sthirugn — APPROVED** [review]

**Pull request merged**
